public/app/plugins/datasource/prometheus/components/PrometheusMetricsBrowser.test.tsx

```diff
@@ -114,7 +114,13 @@ describe('PrometheusMetricsBrowser', () => {
         }
         return [];
       },
-      fetchLabelsWithMatch: (selector: string) => {
+      // This must always call the series endpoint 
+      // until we refactor all of the metrics browser
+      // to never use the series endpoint.
+      // The metrics browser expects both label names and label values.
+      // The labels endpoint with match does not supply label values 
+      // and so using it breaks the metrics browser.
+      fetchSeriesLabels: (selector: string) => {
```

Comment:
❤️

Reply:
```suggestion
      // This must always call the series endpoint
      // until we refactor all of the metrics browser
      // to never use the series endpoint.
      // The metrics browser expects both label names and label values.
      // The labels endpoint with match does not supply label values
      // and so using it breaks the metrics browser.
      fetchSeriesLabels: (selector: string) => {
```

Reply:
I did `prettier:check` and `prettier:write` this was what it was complaining about. Just the spaces and the end of the lines.